apps/cli/src/node/docker-compose-dev.yaml

```diff
@@ -158,15 +158,28 @@ services:
         depends_on:
             validator:
                 condition: service_healthy
+            traefik-config-generator:
+                condition: service_completed_successfully
         healthcheck:
-            test: ["CMD", "traefik", "healthcheck"]
+            test: ["CMD", "traefik", "healthcheck", "--ping"]
             interval: 5s
             timeout: 1s
             retries: 5
+        command:
+            [
+                "--ping=true",
+                "--entryPoints.web.address=:8088",
+                "--entryPoints.traefik.address=:8080",
```

Comment:
During the `PING health check`, we need to assign the traefik address (management port) to `8080` because there is no option to define any ports specific (actually there is and I have tested it it did not work) for the `healthcheck ping command` and the default port for it is `8080` for it. 

That's why I have changed the management port and the web-address port locally. 

But, changed the docker compose mapping port to work as same as before. (localhost:8080 ---> web-addresses)